### DISCUSSION TITLE
Limitation to 20 items in favorites

### DISCUSSION THREAD

**Discussion by PulsarFactory** [jellyfin/jellyfin#8508] [Q&A]
Hello,

I have a simple question about favorites in home page of Jellyfin. Favorites seems to be limited to 20 items for each category (films/music/series etc...). 

When i consult logs i find this request : Items?sortOrder=Ascending&Filters=IsFavorite&**Limit=20**&IncludeItemTypes=Movie

Does anyone know why there’s a limitation ?

Thanks.

**Comment by pypp** [comment]
> Hello,
> 
> I have a simple question about favorites in home page of Jellyfin. Favorites seems to be limited to 20 items for each category (films/music/series etc...).
> 
> When i consult logs i find this request : Items?sortOrder=Ascending&Filters=IsFavorite&**Limit=20**&IncludeItemTypes=Movie
> 
> Does anyone know why there’s a limitation ?
> 
> Thanks.

Probably to not load  all your favorites at once.
